ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/network_services_key.schema.yml

```diff
@@ -0,0 +1,1000 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+dynamic_keys:
+  "network_services_keys.name":
+    type: list
+    primary_key: name
+    convert_types:
+      - dict
+    items:
+      type: dict
+      keys:
+        name:
+          type: str
+          required: true
+          description: |
+            Specify a tenant name.
+            Tenant provide a construct to group L3 VRFs and L2 VLANs.
+            Networks services can be filtered by tenant name.
+        mac_vrf_vni_base:
+          type: int
+          convert_types:
+            - str
+          min: 1000
+          max: 16770000
+          description: |
+            Base number for MAC VRF VXLAN Network Identifier (Required with VXLAN)
+            VXLAN VNI is derived from the base number with simple addition.
+            e.g. mac_vrf_vni_base = 10000, svi 100 = VNI 10100, svi 300 = VNI 10300.
+        mac_vrf_id_base:
+          type: int
+          convert_types:
+            - str
+          min: 1000
```

Comment:
change min value to 10000

Reply:
there is actually not a minimum here. It depends on how other network services are provisioned, so we cannot enforce a hard limit. We should have 0 as minimum.

Reply:
just for this key or mac_vrf_vni_base also?